frontend/src/layout/navigation/TopBar/notificationsLogic.tsx

```diff
@@ -55,9 +55,9 @@ export const notificationsLogic = kea<notificationsLogicType>([
                     clearTimeout(values.pollTimeout)
 
                     try {
-                        const response = (await api.get(
+                        const response = await api.get(
                             `api/projects/${teamLogic.values.currentTeamId}/activity_log/important_changes`
-                        )) as ChangesResponse
+                        )
```

Comment:
I think get can take a type right? like `api.get<ChangesResponse>(`

Reply:
Good catch. This is fairly insignificant, because knowledge of this type isn't used in the function, and the return type is already specified because of the reducer's type – but better to retain the type info, so I made use of generics as you suggested